model/database/spans.yaml

```diff
@@ -30,6 +30,7 @@ groups:
         sampling_relevant: true
       - ref: db.operation.batch.size
       - ref: db.response.returned_rows
+        requirement_level: opt_in
```

Comment:
yay for policy checks! (non-stable attributes must be opt-in on stable conventions)